app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -536,6 +570,19 @@ void onItemLoaded(@NonNull Item response) {
         bindContent();
     }
 
+    private void downloadFileAndRenderPdf() {
+        mFileDownloader.downloadFile(mItem.getUrl(), PDF_MIME_TYPE, new FileDownloader.FileDownloaderCallback() {
+            @Override
+            public void onFailure(IOException e) {
+            }
```

Comment:
Should we now fall back to the flow where we prompt user to open it in another app? Or at least show a toast?